src/find/azext_find/custom.py

```diff
@@ -46,8 +48,27 @@ def process_query(question):
                 print('\033[1m' + current_title + '\033[0m')
                 print(current_snippet)
 
-                if i + 1 < num_results_to_show:
-                    print("\n")
+                print("")
+            feedback = prompt("[\033[1mEnter to close.\033[0m Press + or - to give feedback]:")
+            if feedback in ['+', '-']:
+                print('Wow, you are a true hero!')
+                print("""\
+        O_
+       \033[1m<T>\033[0m`-.
+        \033[1m|\033[0m`-‘
+        \033[1mI\033[0m
+                        """)
+                print('My human overloads review each of these reports; I\'m told these reports makes me smarter.')
```

Comment:
`human overlords`?

Reply:
@JasonRShaver @albaloo  Also, was this vetted through anyone? I'm tempted approve, but I want to make sure this doesn't give off an unintended impression of our product.